docs/NooBaaNonContainerized/NooBaaCLI.md

```diff
@@ -59,14 +59,20 @@ The `account add` command is used to create a new account with customizable opti
 
 #### Usage
 ```sh
-noobaa-cli account add --name <account_name> --uid <uid> --gid <gid> [--user]
+noobaa-cli account add --name <root_account_name> --uid <uid> --gid <gid> [--user]
```

Comment:
You added something that we didn't had before - the ability to create IAM accounts using the CLI.

Reply:
I'm not following.
Is that a bad thing?
Is this not what we wanted?